contrib/node/src/python/pants/contrib/node/subsystems/node_distribution.py

```diff
@@ -144,23 +131,22 @@ def install_node(self):
     :returns: The Node distribution bin path.
     :rtype: string
     """
-    node_package_path = self.unpack_package(
-      supportdir=self._supportdir, version=self.version, filename='node.tar.gz')
+    node_package_path = self.select()
```

Comment:
should we have `context` here?

Reply:
No, because we didn't set replaces_scope/replaces_name, so it's not needed. And the fewer places we plumb it through the fewer places we have to remove it from when the deprecation is over.